### DISCUSSION TITLE
七夕将至！

### DISCUSSION THREAD

**Discussion by FlameF6** [share121/inter-knot#253] [其他]
[[兄弟们，距离2024年七夕还有不到40分钟，但是我却没有一个可以陪我一起过七夕的人（当然希望是异性♀，就算有同性♂来找我玩我也不去）。啊啊啊落魄了啊啊啊
![Screenshot_20220724_165913_com ss android ugc aweme_edit_135593799815246](https://github.com/user-attachments/assets/53329629-060b-4bc1-8126-b62e9b782d5c)

**Comment by Luckyxinchen** [comment]
当垃圾小浣熊也挺好的😀
